### ISSUE TITLE
[Discussion] Web of Trust?

### ISSUE THREAD

**Issue by Error-418** [henryboldi/felony#43]
How does this affect web of trust? Can I sign a friend's key with my key to add to their trust level? Or is it like Signal, were the trust is just implicit and I have to hope that my buddy remembers to call or visit me when we swap keys?

**Comment by Fastidious** [comment]
I have not installed it (I need to be able to import my existing keys, not generate new ones), but reading the README it seems it does encrypt and decrypt, but no key signing.

**Comment by xornand** [comment]
@Fastidious nah, the API says otherwise. https://keybase.io/docs/api/1.0/call/merkle/root I'm not sure if this is the conventional Web of Trust (i.e., a la PGP's WoT with Alice signs Bob at a certain trust level, then optionally uses the defacto-standard MIT pgp.keyserver.mit.edu or whatever (is that server still up even?)), but there's at least the concept of both WoT and 'signing' -- both in the sense of encrypt/sign-the-message [in this PGP standard way](https://www.gnupg.org/gph/en/manual/r606.html), as well as the "merkle" concept of endorsing someone as evidenced by the API.

**Comment by Fastidious** [comment]
@xornand oh, Felony is using keybase.io API? Did not know, neat!

**Comment by xornand** [comment]
@Fastidious sorry, I was confusing two threads (didn't have my coffee yet). Nope it doesn't yet but it's on my list of stuff to add as soon as I either get a Mac or block in some GUI to replace the non-existent functionality on the *nix / Win versions
